src/nvim/testdir/test_arglist.vim

```diff
@@ -26,6 +30,8 @@ func Test_argidx()
 endfunc
 
 func Test_argadd()
+  call Reset_arglist()
```

Comment:
I don't see the rationale in the commit message and it's not in Vim.
What's the issue?

Reply:
I suppose that this is to match `Test_zero_argadd`.

Reply:
```
  From test_command_count.vim:
  Found errors in Test_command_count_2():
  function RunTheTest[37]..Test_command_count_2 line 9: Expected 'a' but got ''
```

I think arglist reset is needed.